etcdserver/etcdhttp/client_auth_test.go

```diff
@@ -25,7 +26,17 @@ import (
 	"github.com/coreos/etcd/etcdserver/auth"
 )
 
-const goodPassword = "$2a$10$VYdJecHfm6WNodzv8XhmYeIG4n2SsQefdo5V2t6xIq/aWDHNqSUQW"
+const goodPassword = "good"
```

Comment:
This is changed because we use the fake functions to generate hashed password from the original one, so the good hashed password is changed.